api/config/shared/global.go

```diff
@@ -133,12 +133,35 @@ func (c *GlobalConfig) Validate() error { // nolint gocyclo
 				}
 			}
 		}
+	case "gcs":
+		if bu.GetGcs().GetBucket().GetProjectid().GetValue() == "" {
+			cfgErr.AddMissingKey("global.v1.backups.gcs.bucket.projectid")
+		}
+
+		if bu.GetGcs().GetBucket().GetName().GetValue() == "" {
+			cfgErr.AddMissingKey("global.v1.backups.gcs.bucket.name")
+		}
+
+		// Make sure the user has the path to the gcs creds set and it is accessible
+		gcscredpath := bu.GetGcs().GetCredentials().GetPath().GetValue()
+		if gcscredpath == "" {
+			cfgErr.AddMissingKey("global.v1.backups.gcs.credentials.path")
+		} else {
+			ok, err := fileutils.Readable("hab", gcscredpath)
```

Comment:
this seems problematic. are you manually creating the hab user and group before installing automate?

Reply:
I'm not manually creating the user, I do believe it automatically gets created when you install chef-automate

Reply:
no longer using credentials.path